lcc/models.py

```diff
@@ -662,6 +664,12 @@ def parent_classifications(self):
         return settings.TAXONOMY_CONNECTOR.join(
             self.legislation.classifications.values_list('name', flat=True))
 
+    def save(self, *args, **kwargs):
+        match = re.search('\d+', self.code)
+        if match:
```

Comment:
Maybe we could use a tracker and do the number update only when the article code changes?

I assume articles are not going to be changed a lot, but at least we avoid doing the work twice in the above migration.